src/debugging/netcore/NetCoreDebugHelper.ts

```diff
@@ -285,7 +285,11 @@ export class NetCoreDebugHelper implements DebugHelper {
         }
 
         const debuggerPath: string = '/remote_debugger';
-        const installDebugger: string = `/bin/sh -c "ID=default; if [ -e /etc/os-release ]; then . /etc/os-release; fi; echo $ID; if [ $ID == alpine ]; then apk --no-cache add curl && curl -sSL https://aka.ms/getvsdbgsh | /bin/sh /dev/stdin -v latest -l ${debuggerPath} ; else apt-get update && apt-get install unzip && curl -sSL https://aka.ms/getvsdbgsh | /bin/sh /dev/stdin -v latest -l ${debuggerPath}; fi"`
+        // Windows require double quotes and Mac and Linux require single quote.
+        const osProvider = new LocalOSProvider();
+        const installDebugger: string = osProvider.os === 'Windows' ?

```

Comment:
Would prefer `os.platform() === 'win32'` to be consistent with the other newer code on tasks/debugging.

Reply:
I was following this debugging code https://github.com/microsoft/vscode-docker/blob/aa268845b800e2d833dc03dbc8eb23b2c8cb97e7/src/debugging/coreclr/debuggerClient.ts#L40.